scrapers/nytData.js

```diff
@@ -0,0 +1,36 @@
+const axios = require('axios');
+const csv = require('csvtojson');
+const logger = require('../utils/logger');
+const { updateCache } = require('../utils/nyt_cache');
+
+const US_COUNTY_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-counties.csv';
+const US_STATE_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us-states.csv';
+const US_NATION_WIDE_DATA_URL = 'https://raw.githubusercontent.com/nytimes/covid-19-data/master/us.csv';
```

Comment:
cant the data ulrs be represented as a base and then having their ending appended respectively?

`base = "https://raw.githubusercontent.com/nytimes/covid19-data/master/"`